srcpkgs/fcft/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'fcft'
+pkgname=fcft
+version=2.2.2
+revision=1
+wrksrc=$pkgname
+build_style=meson
+hostmakedepends="pkg-config scdoc"
+makedepends="fontconfig-devel freetype-devel pixman-devel tllist"
+short_desc="Simple library for font loading and glyph rasterization"
+maintainer="Isaac Freund <ifreund@ifreund.xyz>"
+license="MIT"
+homepage="https://codeberg.org/dnkl/fcft"
+distfiles="${homepage}/archive/${version}.tar.gz"
```

Comment:
`distfiles+=" ${homepage}/raw/branch/master/LICENSE"`, `skip_extraction=LICENSE` and `vlicense ${XBPS_SRCDISTDIR}/${pkgname}-${version}/LICENSE` until the next release to make sure we are grabbing the license text from upstream.

Reply:
Good call, done.